### ISSUE TITLE
Triangle-Triangle intersection technique

### ISSUE THREAD

**Issue by Kevinktv** [dilevin/computer-graphics-bounding-volume-hierarchy#107]
I saw this:
https://github.com/dilevin/computer-graphics-bounding-volume-hierarchy/issues/90

And just wanted to confirm whether my strategy is right or not.

Basically just compute edge rays for one triangle and then see whether those edges intersect the other triangle using ray-triangle intersection. (Ensuring 0<t<1 using min_t and max_t so the ray is technically a line segment). 

Is this what you meant in tutorial?

Thanks!

**Comment by Zhecheng-Wang** [comment]
Yes. That is correct. Hint: think about how many rays you will need.

**Comment by Pranjal-R-Agrawal** [comment]
Make sure that your ray-triangle intersection test is able to handle coplanar rays and triangles.
